File: js/p010_sumPrimes.js
/**
Code by Gonzalo Rolon Morinelli, Dec 16 2015.

The following code was originally written as a solution for Project Euler's
Problem# 10
For more information, goto: https://projecteuler.net/problem=10
*/

var primesImport = require('./p003_primes.js');

if (require.main === module) {
    // Problem solution.
    console.log(primesImport.getPrimesLt(2000000).reduce(function(a, b) {
        return a + b;
    }));
}

File: python/p010_sumPrimes.py
"""
Code by Gonzalo Rolon Morinelli, Sep 21 2015.

The following code was originally written as a solution for Project Euler's
Problem# 10.
For more information, goto: https://projecteuler.net/problem=10
"""

import p003_primes as primes_import

if __name__ == '__main__':
    # Problem solution
    print sum(primes_import.get_primes_lt(2000000))
